src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -0,0 +1,188 @@
+.. ## Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _tpls-label:
+
+*********************
+Third-party Libraries
+*********************
+
+Axom dependencies are grouped into four categories: Git submodules,
+built-in TPLs in the Axom source tree, system-level TPLs, and other 
+TPL libraries. The following sections describe how to install and update 
+these dependencies for Axom.
+
```

Comment:
It might be helpful to have a transition sentence. E.g. 

> The following sections describe how to install and update these dependencies for Axom. It also answers questions like:

Reply:
I am going to delete that section once I answer all those.. they were on the top of the old page.